rest_framework_serializer_extensions/serializers.py

```diff
@@ -288,7 +292,8 @@ def get_expand_id_field(self, field_name, field_definition):
         if hasattr(self, 'get_{0}_id'.format(field_name)):
             return serializers.SerializerMethodField(source='*')
 
-        kwargs = dict(read_only=True)
+        read_only = field_definition.get('read_only', True)
```

Comment:
I think it would be nice to put this in `_standardise_expandable_definition()` so that we can have some assurance that `read_only` exists. Ultimately I think we should make an `ExpandableField` class to make the interface slightly nicer.